### ISSUE TITLE
[Bug]: Open at boot not working as expected after Moment 4 update

### ISSUE THREAD

**Issue by onthejobtraining** [TranslucentTB/TranslucentTB#936]
### Please check for a duplicate first

- [X] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

1. Enable startup or 'Open at boot' for TranslucentTB
2. Shutdown or restart your windows device running moment 4 update

### Expected results

TranslucentTB should open and start functioning normally at boot

### Actual results

It doesn't work as expected at boot. This has started to happened after updating to Moment 4.
OS: Windows 11 Home
Build: 22621.2361

### Windows version

Windows 11 SV2 (builds 22621-22622)

**Comment by Univers4craft** [comment]
Hello, I am French, so I apologize for my approximate English. Here are the steps that personally resolved my issue:

1 Uninstall the application.

2 Delete all temporary files without exception from the control panel.

3 Restart pc

4 Reinstall the program, and it should work.

**Comment by metronightly** [comment]
> Hello, I am French, so I apologize for my approximate English. Here are the steps that personally resolved my issue:
> 
> 1 Uninstall the application.
> 
> 2 Delete all temporary files without exception from the control panel, as shown in the photo if necessary. ![Screenshot 2023-10-12 204304](https://user-images.githubusercontent.com/32676828/274701139-cc98ecbe-5e9a-472e-bd2a-1759ce8ffc6f.png)
> 
> 3 Restart pc
> 
> 4 Reinstall the program, and it should work.

Thanks, but it didn't work for me. However, you should remove the image above because it contains sensitive information 😅

**Comment by onthejobtraining** [comment]
> 4 Reinstall the program, and it should work.

Thanks, but it didn't work for me. However, you should remove the image above because it contains sensitive information 😅

**Comment by sylveon** [comment]
Do you get an error at startup

**Comment by onthejobtraining** [comment]
> Do you get an error at startup

No, I am not getting any error on startup. But it works fine if I launch the app manually.
